editoast/editoast_schemas/src/paced_train.rs

```diff
@@ -24,4 +48,92 @@ pub struct PacedTrain {
     pub train_schedule_base: TrainSchedule,
     #[schema(inline)]
     pub paced: Paced,
+    #[serde(default = "Vec::new")]
+    pub exceptions: Vec<PacedTrainException>,
+}
+
+#[derive(Debug, Clone, Default, Serialize, Deserialize, PartialEq, ToSchema)]
+pub struct PacedTrainException {
+    pub id: String,
+    #[schema(inline)]
+    #[serde(flatten)]
+    pub exception_type: PacedTrainExceptionType,
+    pub train_name: Option<TrainNameSilo>,
+    pub rolling_stock: Option<RollingStockSilo>,
+    pub labels: Option<LabelsSilo>,
+    pub category: Option<CategorySilo>,
+    #[schema(inline)]
+    pub speed_limit_tag: Option<SpeedLimitTagSilo>,
+    pub start_time: Option<StartTimeSilo>,
+    pub constraint_distribution: Option<ConstraintDistributionSilo>,
+    pub initial_speed: Option<InitialSpeedSilo>,
+    pub options: Option<OptionsSilo>,
+    pub path_and_schedule: Option<PathAndScheduleSilo>,
+}
+
+#[derive(Debug, Clone, Default, Serialize, Deserialize, PartialEq, Eq, ToSchema)]
+#[serde(tag = "type")]
+pub enum PacedTrainExceptionType {
+    #[default]
+    Created,
+    Modified {
```

Comment:
Are we sure this the only modification kind we'll ever support? If not I'd suggest
```suggestion
    Disabled {
```

Reply:
I’m not sure about ‘Disabled’, because we have two types of exceptions: one is a newly created exception that is not part of the occurrences, and the other is a modified occurrence that differs from the others. The ‘disabled’ field only represents the state of a specific occurrence.

Reply:
Indeed, to me "disabled" is a subset of "modified".

Reply:
I didn't understand the meaning of the schema, so yes the `Modified` name is fine with me. But by looking at its fields for the first time without context or docuemntation, it screams "which occurence_id do I disable?". ("Disabled" is indeed a subset of "modified", so let's be specific when we can.)

Since the schema is not explicit by itself, imo an even better move than documenting it would be to change it so that its structure explains on its own what the schema means. `enum { Created, Modified { occurrence_id: usize } }` is straightforward. Hence my proposition to move the `disabled` at the `PacedTrain` level.